packages/features/data-table/components/DataTable.tsx

```diff
@@ -39,6 +39,7 @@ export type DataTablePropsFromWrapper<TData> = {
   headerClassName?: string;
   rowClassName?: string;
   paginationMode?: "infinite" | "standard";
+  hasWrapperContext?: boolean;
```

Comment:
We need this due to the fact wrapper adds a lot of layout changes. i.e toolbar etc. So this allows us to have the same style in both useages where data-table wrapper is not used

Reply:
good addition!